src/index.ts

```diff
@@ -86,13 +90,25 @@ export async function init(config: IClientConfig): Promise<IEppoClient> {
     requestTimeoutMs: config.requestTimeoutMs ?? undefined,
     numInitialRequestRetries: config.numInitialRequestRetries ?? undefined,
     numPollRequestRetries: config.numPollRequestRetries ?? undefined,
-    pollAfterSuccessfulInitialization: true, // For servers we always want to keep polling for the life of the server
+    pollAfterSuccessfulInitialization: true, // For servers, we always want to keep polling for the life of the server
     pollAfterFailedInitialization: config.pollAfterFailedInitialization ?? false,
     throwOnFailedInitialization: config.throwOnFailedInitialization ?? true,
   };
 
-  clientInstance = new EppoClient(configurationStore, requestConfiguration);
-  clientInstance.setLogger(config.assignmentLogger);
+  const flagConfigurationStore = new MemoryOnlyConfigurationStore<Flag>();
+  const banditVariationConfigurationStore = new MemoryOnlyConfigurationStore<BanditVariation[]>();
```

Comment:
This store is needed to easily know if a flag has any bandits, which we want for handling an edge case of empty actions being passed to `getBanditAction()`